content/applications/finance/accounting/others/adviser/budget.rst

```diff
@@ -2,156 +2,101 @@
-:menuselection:`Advisers --> Analytic Accounts --> Open Charts`. Create a new Account
-called Smith&Co project and select the related partner.
+When creating a new customer invoice and/or vendor bill, you have to refer to this analytic account.
 
-.. image:: budget/budget06.png
+.. image:: budget/analytic-accounts.png
    :align: center
+   :alt: add analytic accounts in a new invoice or bill.
 
-Set a budget
-============
+Define the budget
+-----------------
 
-Let's now set our targets for our budget. We specified that we expect to
-gain 1000 with this project and we would like not to spend more than
-700.
+Let’s set our targets. We specified that we expect to gain 1000 with this project, and we would like
+not to spend more than 700. Go to :menuselection:`Accounting --> Management: Budgets` and click
+:guilabel:`New` to create a new budget for *Smith & Co* project.
 
-To set those targets, enter the accounting app, select :menuselection:`Advisers -->
-Budgets` and create a new Budget.
+First, fill in your :guilabel:`Budget Name`. Then, select the :guilabel:`Period` wherein the budget
+is applicable. Next, add the :guilabel:`Budgetary Position` you want to track, define the related
+:guilabel:`Analytic Account`, and add the :guilabel:`Planned Amount`.
 
-We have to give a name to the budget. In this case we'll call it "Smith
-Project". Select the period wherein the budget will be applicable. Next
-add an item to specify your targets in the Budget Line.
-
-.. image:: budget/budget07.png
+.. image:: budget/define-the-budget.png
    :align: center
+   :alt: budget lines display
 
-Select the Budgetary Position related to the Budget Line. In other
-words, select the position that points to the accounts you want to
-budget. In this case we will start with our 700 maximum charge target.
-Select the "Cost" Budgetary Position and specify the Planned Amount.
-As we are recording a cost, we need to specify a **negative amount**.
-Finally, select the corresponding analytic account.
-
-.. image:: budget/budget08.png
-   :align: center
-
-Click on **Save & new** to input the revenue budget. The Budgetary
-Position is Revenue and the Planned Amount is 1000. Save and close
-
-You'll need to **Confirm** and **Approve** the budget.
+.. Note::
+   When recording a planned amount related to expenses, the amount must be negative.
 
 Check your budget
-=================
+-----------------
 
-You can check your budget at any time. To see the evolution, let's book
-some Invoices and Vendors Bills.
+Go to :menuselection:`Accounting --> Management: Budgets` and find the *Smith & Co* Project to see
+how it evolves according to the expenses or income for the related analytic account.
 
-.. tip::
-   If you use analytical accounts remember that you need to specify the account in the invoice
-   and/or purchase line.
+The :guilabel:`Practical Amount` evolves when a new journal entry related to an account from your
+budgetary position and your analytic account is created.
```

Comment:
en ref au training de fp, I'd switch the two groups to make the sentence clearer (ps: did not check with PyCharm for the # of characters)
```suggestion
The :guilabel:`Practical Amount` evolves when a new journal entry related to your analytic account and an account from your budgetary position is created.
```